src/Cmts.ml

```diff
@@ -26,7 +26,7 @@ type t =
     whenever 2 intervals share more than an end-point, then one contains the
     other. *)
 module Non_overlapping_interval_tree (Itv : sig
-  include Hashtbl.Key
+  include Hashtbl.Key[@@ocaml.warning "-3"]
```

Comment:
Tangential, but is it intended that there is no space before the attribute here?

Reply:
I'll let @Julow or @gpetiot take a look.

Reply:
https://github.com/ocaml-ppx/ocamlformat/pull/775